privacyidea/lib/policy.py

```diff
@@ -2444,6 +2445,10 @@ def get_static_policy_definitions(scope=None):
                 'desc': _("This action adds a QR code in the enrollment page for "
                           "HOTP, TOTP and Push tokens, that lead to this given URL."),
                 'group': 'QR Codes'
+            },
+            ACTION.TOKENINFO_UI_EXCLUDE: {
```

Comment:
You are putting this in the scope "webui", although the information is removed from the API level. I would choose another scope, like admin or user. Since it is somehow an access right to see certain information.

Reply:
Okay I will move it to the scope `user` my thought was that it has an effect over the UI so I included it under `webui`